docs/menu.yml

```
@@ -318,6 +320,8 @@ catalog:
         path: "/en/concepts-and-designs/lal"
       - name: "Profiling"
         path: "/en/concepts-and-designs/profiling"
+      - name : "Configure Service Hierarchy"
+        path: "/en/concepts-and-designs/configure-service-hierarchy"
```

```suggestion
      - name : "Service Hierarchy Configuration"
        path: "/en/concepts-and-designs/service-hierarchy-configuration"
```

Could you rename the file and menu as noun, instead starting from a verb.